app/_components/MutatedGeneCNATable.js

```diff
@@ -578,16 +579,17 @@ window.EnhancedFixedDataTableSpecial = (function() {
 
     // Creates Qtip after page scrolling
     onScrollEnd: function() {
-      $(".qtip").remove();
+      var qtipId = '.qtip';
+      if (this.props.elementId) {
+        qtipId = '.' + this.props.elementId + '-qtip ' + qtipId;
+      }
+      $(qtipId).remove();
```

Comment:
what if `this.props.elementId` is `undefined`? it runs `$(".qtip").remove();` would that be ok?

Reply:
Currently I set the property default elementId to empty String. What about I set it to `EFDT`? It would be better only remove qtip in EFDT tables?